server/controllers/appointment.js

```diff
@@ -0,0 +1,65 @@
+import express from 'express';
+import model from '../models';
+
+export default {
+  postappointment: async (req, res) => {
+    // console.log('my user', req.user);
+    // console.log('health', req.body.Healthprofessional);
+    try {
+      const postAppoint = await model.Appointments.create({
+        message: req.body.message,
+        ClientId: req.user.clientData.id,
+        Healthprofessional: req.body.Healthprofessional,
+      });
+      //   console.log('appoint', postAppoint);
+      res.send({
+        postAppoint,
+        message: 'your appointment was successful',
+      });
+    } catch (error) {
+      //   return res.status(500).send({ message: error.message });
```

Comment:
Uncomment the error message

Reply:
After creating an appointment, are the users updated on this

Reply:
The user is not updated, I am only grabbing the logged-in user-id to be able to make this request , if no user u can't request for appointment

Reply:
noted.... uncomment issues